contracts/mocks/extensions/tokenProperties/RMRKTokenPropertiesMock.sol

```diff
@@ -2,29 +2,25 @@
 
 pragma solidity ^0.8.16;
 
-import "./IRMRKTokenProperties.sol";
+import "../../../RMRK/extension/tokenProperties/IRMRKTokenProperties.sol";
 
 /**
  * @title RMRKTokenProperties
  * @author RMRK team
  * @notice Smart contract of the RMRK Token properties module.
  */
-contract RMRKTokenProperties is IRMRKTokenProperties {
+contract RMRKTokenPropertiesMock is IRMRKTokenProperties {
 
-    uint256 private totalStringProperties;
-    mapping(uint256 => string) stringProperties;
-    mapping(uint256 => mapping(string => uint256)) private _stringProperties;
+    mapping(string => uint256) private _keysToIds;
+    uint256 private _totalProperties = 0;
 
-    uint256 private totalAddressProperties;
-    mapping(uint256 => address) addressProperties;
-    mapping(uint256 => mapping(string => uint256)) private _addressProperties;
+    mapping(string => uint256) private _stringValuesToIds;
+    mapping(uint256 => mapping(uint256 => string)) private _stringValues;
```

Comment:
This seems different from what we agreed on the call 🤔 

I think this is in line with what we agreed on:

```suggestion
    mapping(uint256 => string) private dictionary;
    mapping(string => uint256) private reverseDictionary;
    mapping(uint256 => mapping(uint256 => uint256)) private _stringValues;
```